src/components/table/arrTable.js

```diff
@@ -1,9 +1,9 @@
 const arrTableRow = [
-    {rowTitle: 'Design: A Survival Guide for Beginners', rowDate: 'Posted 3 days ago', rowStatus: 'published', rowViews: 120},
-    {rowTitle: 'Design: A Survival Guide for Beginners', rowDate: 'Posted 3 days ago', rowStatus: 'draft', rowViews: 0},
-    {rowTitle: 'Design: A Survival Guide for Beginners', rowDate: 'Posted 3 days ago', rowStatus: 'scheduled', rowViews: 0},
-    {rowTitle: 'Design: A Survival Guide for Beginners', rowDate: 'Posted 3 days ago', rowStatus: 'published', rowViews: 120},
-    {rowTitle: 'Design: A Survival Guide for Beginners', rowDate: 'Posted 3 days ago', rowStatus: 'published', rowViews: 120},
+    {title: 'Design: A Survival Guide for Beginners', date: 'Posted 3 days ago', status: 'published', views: 120},
```

Comment:
if you want display field such as this `date` you need to take the date of creating and calculate typically string